src/components/Nav/Mobile.tsx

```diff
@@ -230,56 +136,89 @@ const MobileNavMenu: React.FC<IProps> = ({
       gap={4}
       sx={{ svg: { fill: "body" } }}
     >
-      <IconButton
-        icon={<Icon name="search" />}
+      <SearchIconButton
         onClick={toggleSearch}
         aria-label={t("aria-toggle-search-button")}
-        variant="icon"
-        _hover={{ svg: { fill: "primary" } }}
       />
       <IconButton
         icon={
-          <GlyphButton
+          <Icon
             viewBox="0 0 24 40"
             pointerEvents={isMenuOpen ? "none" : "auto"}
+            mx={0.5}
+            width={6}
+            height={10}
+            position="relative"
+            strokeWidth="2px"
+            zIndex={100}
+            _hover={{
+              color: "primary",
+              "& > path": {
+                stroke: "primary",
+              },
+            }}
+            sx={{
+              "& > path": {
+                stroke: "text",
+                fill: "none",
+              },
+            }}
           >
             <motion.path
               variants={glyphPathVariants}
               initial={false}
               animate={isMenuOpen ? "open" : "closed"}
             />
-          </GlyphButton>
+          </Icon>
```

Comment:
This icon is not correctly placed within the `Drawer`.
![image](https://user-images.githubusercontent.com/468158/232620742-6e3213e1-ce57-4080-9a79-824248d08e14.png)

I think the `Drawer` has a higher `zIndex` than the button.

Reply:
Oh no!

The Drawer renders in a portal, but I know of a trick to get this working. 😉